octoversion.json

```
@@ -1,3 +1,3 @@
 {
-    "NonPreReleaseTags": ["refs/heads/main"]
+    "NonPreReleaseTags": ["refs/heads/main", "main"]
```

```suggestion
    "NonPreReleaseTags": ["main"]
```

We shouldn't need that once we are only building here in Actions.

---

Lols. I just asked Edlyn to add it :)